duckduckgo/iOS
# add cookie debug screen and diagnostic report generation

Task/Issue URL: https://app.asana.com/0/392891325557410/1200508873958867
Tech Design URL:
CC:

**Description**:

Add a new cookie debug screen and allow user to generate a diagnostics report.

Also small refactoring to clear cookies more effectively. 

**Steps to test this PR**:
1. Generally use the app, then go to settings > version > and click on the share icon.  
1. Use the activity sheet to save the report. 
1. Check it was created correctly and looks valid.
1. Use the Fire Button.
1. Ensure cookies cleared
1. Generate another report and make sure it reflects the new state.



**Device Testing**:


**OS Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
